src/utils/static-eval.js

```diff
@@ -151,7 +151,7 @@ const visitors = {
     let fn = callee.value;
     if (typeof fn === 'object' && fn !== null) fn = fn[FUNCTION];
     if (typeof fn !== 'function') return;
-    
+
     const ctx = node.callee.object && walk(node.callee.object).value || null;
```

Comment:
I just noticed that this `walk(node.callee.object).value` case would throw if there was an undefined return from walk. That would throw the entire build, so probably worth fixing this.

Reply:
Fixed this, I think I encountered this bug at some point while I was working with the `WILDCARD` implementations but chalked it off as something I did wrong.

Reply:
Great.